pkg/views/server/configure.go

```diff
@@ -113,23 +134,75 @@ func ConfigurationForm(config *apiclient.ServerConfig, containerRegistries []api
 		huh.NewGroup(
 			huh.NewInput().
 				Title("Frps Domain").
-				Value(config.Frps.Domain),
+				Value(m.config.Frps.Domain),
 			huh.NewInput().
 				Title("Frps Port").
 				Value(&frpsPortView).
-				Validate(createPortValidator(config, &frpsPortView, config.Frps.Port)),
+				Validate(createPortValidator(m.config, &frpsPortView, m.config.Frps.Port)),
 			huh.NewInput().
 				Title("Frps Protocol").
-				Value(config.Frps.Protocol),
+				Value(m.config.Frps.Protocol),
 		),
-	).WithTheme(views.GetCustomTheme())
+	)
```

Comment:
Try adding the` .WithTheme(views.GetCustomTheme())` to this line to fix the theme